docs/tutorials/ml_inference/semantic_search/optimize_vector_search_with_cohere_compressed_embedding.md

```diff
@@ -0,0 +1,536 @@
+# Topic
+
+This tutorial is compatible with version 2.19 and later.
+
+This tutorial demonstrates how to optimize vector search by integrating Cohere compression embedding with OpenSearch. Cohere's compression embedding allows for more efficient storage and faster retrieval of vector representations, making it ideal for large-scale search applications.
+
+Bedrock supports compressed embeddings from Cohere Embed now ([blog](https://aws.amazon.com/about-aws/whats-new/2024/06/amazon-bedrock-compressed-embeddings-cohere-embed/)). This tutorial will use Cohere `embed-multilingual-v3` on Bedrock.
+
+We'll use the following OpenSearch features:
+- [ML inference ingest processor](https://opensearch.org/docs/latest/ingest-pipelines/processors/ml-inference/) to generate embedding when ingest data; 
+- [ML inference search request processor](https://opensearch.org/docs/latest/search-plugins/search-pipelines/ml-inference-search-request/)
+- [Search template query](https://opensearch.org/docs/latest/api-reference/search-template/) for searching.
+- [k-NN index](https://opensearch.org/docs/latest/search-plugins/knn/index/) and [k-NN byte vector](https://opensearch.org/docs/latest/field-types/supported-field-types/knn-vector/#byte-vectors)
+
+Note: Replace the placeholders that start with `your_` with your own values.
+
+# Steps
+
+## 1. Create Embedding Model
+
+First, we'll set up a connector to Amazon Bedrock for accessing Cohere's embedding model.
+
+
+### 1.1 Create Connector
+Create a connector by following this [blueprint](https://github.com/opensearch-project/ml-commons/blob/main/docs/remote_inference_blueprints/bedrock_connector_cohere_cohere.embed-multilingual-v3_blueprint.md).
+
+Note: We don't need  pre/post process function in this tutorial as we are going to use ML inference processor.
+
+`"embedding_types": ["int8"]` specifies that we want 8-bit integer quantized embeddings from the Cohere model.
+This meaans embeddings are compressed from 32-bit floats to 8-bit integers.
+It reduces storage space and improves computation speed.
+There's a minor trade-off in precision, but it's usually negligible for search tasks.
+It's compatible with OpenSearch's KNN index, which supports byte vectors.
+
+You can find more details about model parameters on [Cohere docment](https://docs.cohere.com/v2/docs/embeddings).
```

Comment:
Add one more introduction to `truncate` parameter? As it is not in the blog.

Reply:
Will link another document which has all parameters.